Extension/src/nativeStrings.json

```diff
@@ -177,6 +177,9 @@
         "text": "+%d specializations",
         "hint": "Refers to C++ template specializations. This is part of a description indicating there are N additional specializations of a function.  %d is replaced with that number."
     },
+    "compiler": {
+        "text": "Note: Intellisense not fully configured. Use the Select Default compiler command to finish configuration"
```

Comment:
* Should "compiler" be a more specific label?
* "Intellisense" should be "IntelliSense".
* "not" should be "is not".
* "Select Default compiler" should be "Select Default Compiler".
* "configuration" should be "configuration.".